### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.Throwable: Project Project(name=osiris, containerState=DISPOSE_COMPLETED, componentStore=/Users/bandalr/IdeaProjects/osirisMig) already disposed

### ISSUE THREAD

**Issue by BadalRoopRai** [microsoft/azure-tools-for-java#4739]
IntelliJ build version: 2020.2.3 IU-202.7660.26
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.8
Plugin version: 3.44.0-2020.2
Additional Info: None
Parent component: 
```
com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog$DialogRootPane[,0,22,862x478,invalid,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.Throwable: Project Project(name=osiris, containerState=DISPOSE_COMPLETED, componentStore=/Users/bandalr/IdeaProjects/osirisMig) already disposed
	at com.intellij.openapi.diagnostic.Logger.error(Logger.java:143)
	at com.microsoft.intellij.helpers.WhatsNewManager.lambda$createAndShowWhatsNew$1(WhatsNewManager.java:87)
```

**Comment by andxu** [comment]
The complete log is:
```
Caused by: com.intellij.serviceContainer.AlreadyDisposedException: Cannot create com.intellij.openapi.project.DumbService because container is already disposed (container=Project(name=xxxx, containerState=DISPOSE_COMPLETED, componentStore=<REDACTED: user-file-path>) (disposed))
	at com.intellij.serviceContainer.ContainerUtilKt.throwAlreadyDisposedError(containerUtil.kt:43)
	at com.intellij.serviceContainer.ComponentManagerImpl.doGetService(ComponentManagerImpl.kt:612)
	at com.intellij.serviceContainer.ComponentManagerImpl.getService(ComponentManagerImpl.kt:573)
	at com.intellij.openapi.client.ClientAwareComponentManager.getFromSelfOrCurrentSession(ClientAwareComponentManager.kt:37)
	at com.intellij.openapi.client.ClientAwareComponentManager.getService(ClientAwareComponentManager.kt:22)
	at com.intellij.openapi.project.DumbService.getInstance(DumbService.java:197)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.isDumbMode(ActionUtil.java:199)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:110)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.lastUpdateAndCheckDumb(ActionUtil.java:211)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.invokeAction(ActionUtil.java:439)
	at com.microsoft.intellij.AzurePlugin.lambda$initializeWhatsNew$1(AzurePlugin.java:155)
	at com.microsoft.azure.toolkit.lib.common.task.AzureTask.lambda$new$0(AzureTask.java:117)
```

**Comment by wangmingliang-ms** [comment]
fixed, can you please upgrade to v3.60.1
